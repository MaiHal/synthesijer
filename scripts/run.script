@ 2>/dev/null # 2>nul & echo off & goto BOF:
LANG=en_US.UTF-8 LANG=en_US.UTF-8 exec java -jar $JAVA_OPTS "$0" "$@"
exit

:BOF
java -jar %JAVA_OPTS% "%~dpnx0" %*
exit /B %errorlevel%


