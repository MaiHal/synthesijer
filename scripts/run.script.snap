@ 2>/dev/null # 2>nul & echo off & goto BOF:
LANG=en_US.UTF-8 LANG=en_US.UTF-8 exec $SNAP/usr/lib/jvm/java-11-openjdk-amd64/bin/java -jar $JAVA_OPTS "$0" "$@"
exit

:BOF
java -jar %JAVA_OPTS% "%~dpnx0" %*
exit /B %errorlevel%


